doc/source/cluster/kubernetes/user-guides/rayservice.md

```diff
@@ -261,59 +281,7 @@ curl -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc:800
 # [Expected output]: 8
 ```
 
-## Step 9: Why 1 worker Pod isn't ready?
```

Comment:
Ray Serve scheduling strategy changes, so all worker Pods have some Ray Serve replicas. The readiness probes will not fail. We should create a separate doc for it.